CMakeLists.txt

```diff
@@ -162,8 +162,9 @@ endif()
 ################################################################################
 # Installation.
 
+option(alpaka_INSTALL "" OFF)
```

Comment:
please add a short doc instead of `""` e.g.

`allow installation of alpaka if used as project dependency`

Reply:
please add this option additionally to our documentation: https://github.com/alpaka-group/alpaka/blob/develop/docs/source/advanced/cmake.rst